src/Helpers/auth_helper.php

```diff
@@ -43,3 +44,33 @@ function user_id()
         return $auth->id();
     }
 }
+
+if (! function_exists('shieldSetting')) {
+    /**
+     * Provides a wrapper function for settings module.
+     *
+     * @param mixed $value
+     *
+     * @return array|bool|float|int|object|string|void|null
+     * @phpstan-return ($value is null ? array|bool|float|int|object|string|null : void)
+     */
+    function shieldSetting(?string $key = null, $value = null)
+    {
+        /** @var AuthConfig $config */
+        $config = config('Auth');
+        if($config->useSettings) {
+            return setting($key, $value);
+        }
+
+        // Getting the value?
+        if (!empty($key) && count(func_get_args()) === 1) {
+            $parts = explode('.', $key);
+            if (count($parts) === 1) {
```

Comment:
```suggestion
            if (count($parts) !== 2) {
```
How about keeping it this way?